src/com/aca/homework/week4/gun/Gun.java

```diff
@@ -0,0 +1,47 @@
+package com.aca.homework.week4.gun;
+
+public class Gun {
+    int[] cartridges;
```

Comment:
the properties should be private. and there is no need to use an array in this problem